src/Chat/Client/Actions/PostMessageAction.php

```diff
@@ -54,8 +54,9 @@ public function processResponse($response, int $attempt): int
 
         // sometimes we can get {"id":null,"time":null}
         // I think this happens when we repeat ourselves too quickly
+        // todo: remove this if we don't get any more for a week or two (repeat message guard should prevent it)
         $delay = $attempt * 1000;
-        $this->logger->log(Level::DEBUG, "Got a null message post response, waiting for {$delay}ms before trying again");
+        $this->logger->log(Level::ERROR, "WARN: Got a null message post response, waiting for {$delay}ms before trying again");
```

Comment:
note also that the reason I added that WARN is so I can grep the logs easily, I'm hoping I will be able to trash the handler for null response completely if we don't get any for a couple of weeks